peru/runtime.py

```diff
@@ -139,6 +144,67 @@ def print_overrides(self):
             self.display.print('  {}: {}'.format(
                 name, self.get_override(name)))
 
+    def watch_overrides(self):
+        class WatchHandler(LoggingEventHandler):
+            logging.basicConfig(level=logging.INFO,
+                            format='%(asctime)s - %(message)s',
+                            datefmt='%Y-%m-%d %H:%M:%S')
+
+            def on_moved(self, event):
+                super(LoggingEventHandler, self).on_moved(event)
+                what = 'directory' if event.is_directory else 'file'
+                logging.info('moved %s: from %s to %s', what, event.src_path,
+                             event.dest_path)
+                logging.info('running peru sync --force')
+                os.system('peru sync --force')
```

Comment:
Peru doesn't currently do any kind of filesystem locking to prevent running multiple commands in parallel in the same project. But if you do that, it could cause weird conflicts and errors. We should definitely add proper locking at some point, but still also probably makes sense for a watcher to avoid queueing up several waiting instances at once.